kubetest/extract_k8s.go

```diff
@@ -435,12 +456,15 @@ func (e extractStrategy) Extract(project, zone, region string, extractSrc bool)
 		}
 		return getKube(fmt.Sprintf("file://%s", url), release, extractSrc)
 	case gci, gciCi:
-		if i, err := setupGciVars(e.option); err != nil {
+		family, gciExtractParams := parseGciExtractOption(e.option)
```

Comment:
s/gciExtractParams/extraParams/g

Reply:
imo, I think gciExtractParams more clearly describes this variable, as these "params" only belong to the --extract=gci option.